src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends FigCol {
+    private double radius;
+
+    public Circle(double radius, Color color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: " + getArea()
+                + " sq. units, radius: " + radius + " units, color: " + getColor());
```

Comment:
Don't use System.out.println() in the draw() method. Consider using a toString() method and printing the figure details in the main method or another class that handles the user interface.